src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.model.dto.request;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class UserRequestDto {
+    @NotEmpty
+    @Email
```

Comment:
Maybe Email annotation have not null constraint underhood (let’s  check)

Reply:
yeah, also think so, my bad